tutorials/elixir-phoenix-on-kubernetes-google-container-engine/Dockerfile

```diff
@@ -0,0 +1,118 @@
+# Copyright 2017 Google Inc.
```

Comment:
I'm uncertain whether sample code like this should have an Apache license header (when written by a Googler). Does anyone know?

Reply:
The current license is correct. Code in this repository is Apache-2.0 licensed.